mac_os_x.sh

```diff
@@ -1,18 +1,20 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash 
 #
-# This bootstraps Puppet on Mac OS X 10.8 and 10.7.
+# This bootstraps Puppet on Mac OS X 10.9 - 10.12 && is depecrated for 10.7/10.8
 #
 # Optional environmental variables:
 #   - FACTER_PACKAGE_URL: The URL to the Facter package to install.
 #   - PUPPET_PACKAGE_URL: The URL to the Puppet package to install.
 #
 set -e
 
+OSX_VERSION=$(sw_vers | grep -i productversion| awk '{print $2}')
```

Comment:
This looks to return the patch version of OSX as well as the major and minor versions. 
ie: `10.12.6` instead of `10.12`, which is what the puppetlabs download requires. 

Adding `| cut -f1,2 -d'.'`, should solve this.